core/bin/qdrant_migrator.rs

```diff
@@ -345,14 +340,13 @@ async fn migrate_shadow_write(
     loop {
         let now = utils::now();
         let scroll_results = match qdrant_client
-            .scroll(&ScrollPoints {
-                collection_name: ds.qdrant_collection(),
-                with_vectors: Some(true.into()),
-                with_payload: Some(true.into()),
-                limit: Some(points_per_request as u32),
-                offset: page_offset.clone(),
-                ..Default::default()
```

Comment:
Wo we know all the default values that we passed here?

Reply:
They're the qdrant default. We still use these defaults right?

Reply:
Ah yeah we do.